### PR TITLE
Add instructions for interactig with geth testnets

### PR DESCRIPTION
**Pull request by amanusk**
This adds some basic instruction to interacting with available testnets in geth

### CODE REVIEW

File: clients.asciidoc [hunk 1]
```diff
@@ -155,6 +155,7 @@ For other operating systems, use the package manager of your OS or follow the Wi
 
 Now you have +git+, +golang+, +rust+, and necessary libraries installed, let's get to work!
 
+[[Go-Ethereum]]
```
Comment (fjrojasgarcia) [inline clients.asciidoc]: Andreas point out in #279, _"I prefer to use descriptive names with **underscore** between words, up to 2-4 words in length"_.
IMHO, I'd also avoid letter capitalization.

File: ethereum-testnets.asciidoc [hunk 2]
```diff
@@ -23,7 +23,40 @@ Infura was born with the goal of delivering stable and reliable RPC access to th
 https://infura.io/docs/gettingStarted/chooseaNetwork
 
 ==== Geth
-//// TO DO: Loosely explain the geth commands to connect to a testnet
+Geth natively supports both the Ropsten and Rinkeby networks. To connect to the Ropsten network use the command-line argument:
+----
+geth --testnet
+----
+This will start syncing the Ropsten blockchain. A new directory named 'testnet' will be created in you main Ethereum data directory. A 'keystore' directory will be create inside 'testnet' and will store the private keys of you testnet accounts. As of writing this, the Ropsten blockchain is significantly smaller than the main Ethereum blockchain, and consists of about 14GB of data. Since the testnet requires less resouces, it is simpler to setup and test your code on the testnet first.
```
Comment (fjrojasgarcia) [inline ethereum-testnets.asciidoc]: "of you testnet accounts" -> "of **your** testnet accounts"

File: ethereum-testnets.asciidoc [hunk 3]
```diff
@@ -23,7 +23,40 @@ Infura was born with the goal of delivering stable and reliable RPC access to th
...
+
+More on setting up geth at: <<clients.asciidoc#Go-Ethereum,Go-Ethereum (Geth)>>
+
+[TIP]
+====
+Although much smaller, it might still take time for the testnet to fully sync. You can check if syncing was complete by running the following command in the geth intractive console:
+----
+eth.getBlock("latest").number
+----
+This should return a number other than 0 once the your testnet node is fully in sync.
```
Comment (fjrojasgarcia) [inline ethereum-testnets.asciidoc]: "once **the your** testnet node" -> " once your testnet node"

Comment (fjrojasgarcia) [reply ethereum-testnets.asciidoc]: The fact that the last block number != 0 is not enough to conclude that your node is fully synced.
For instance, you could look for the last block number of the network in a known explorer and compare it.

**Review by fjrojasgarcia — CHANGES_REQUESTED** [review]

File: clients.asciidoc (hunk 1 above)
Comment (amanusk) [reply clients.asciidoc]: Good to know, I tried to see if there are any code conventions regarding this but couldn't find any. 
A question, I am trying to reference a paragraph in a different chapter (clients.ascii), this does not seem to work well with regular references. 
Meaning, if I reference something by using `<<refrence>>`, where `[[reference]]` is not in the same file, the reference does not work. 

The workaround I found is to use `<<clients.asciidoc#reference, ParagraphName>>` . Is there a better way to do this, or should I only reference the chapter?

**Comment by fjrojasgarcia** [comment]
Thanks for your contribution @amanusk. Good job!

**Review by fjrojasgarcia — APPROVED** [review]

**Pull request merged**